priv/repo/migrations/20160802030642_create_note.exs

```diff
@@ -0,0 +1,12 @@
+defmodule Notox.Repo.Migrations.CreateNote do
+  use Ecto.Migration
+
+  def change do
+    create table(:notes) do
+      add :title, :string
```

Comment:
Can you create a note without a title? If not, this should be marked `null: false`, so: `add :title, :string, null: false`. While it's possible to add validations to the changeset, you never know if people will go around the changeset later, so if its important for this value to be there, be sure to add the validation.